src/platforms/common/configuration/releases.mdx

```diff
@@ -46,26 +46,44 @@ The value can be arbitrary, but we recommend either of these naming strategies:
 
 <Note>
 
 Releases are global per organization; prefix them with something project-specific for easy differentiation.
 
 </Note>
 
+## Setting a Release
+
 <PlatformContent includePath="set-release" notateUnsupported />
 
-How you make the version available to your code is up to you. For example, you could use an environment variable that is set during the build process.
+How you make the release name (or version) available to your code is up to you. For example, you could use an environment variable that is set during the build process or during initial start-up.
+
+Setting the release name tags each event with that release name. We recommend that you tell Sentry about a new release before sending events with that release name, as this will unlock a few more features. See our documentation about [releases](/product/releases/).
 
-This tags each event with the release value. We recommend that you tell Sentry about a new release before deploying it, as this will unlock a few more features as discussed in our documentation about [releases](/product/releases/). But if you don’t, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
+If you do not tell Sentry about a new release, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
 
 After configuring your SDK, you can install a repository integration or manually supply Sentry with your own commit metadata. Read our documentation about [setting up releases](/product/releases/setup/) for further information about integrations, associating commits, and telling Sentry when deploying releases.
 
 <PlatformSection supported={["apple", "android", "javascript", "flutter", "native", "rust", "python", "react-native", "dotnet", "unity"]}>
 
 ## Release Health
-  
+
+Monitor the [health of releases](/product/releases/health/) by observing user adoption, usage of the application, percentage of [crashes](/product/releases/health/#crash), and [session data](/product/releases/health/#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+In order to monitor release health the SDK sends session data.
+
+### Sessions
+
+A session represents the interaction between the user and the application. Sessions contain a timestamp, a duration, a status (if the session was OK or if it crashed), and other information.
+
+Sessions are always linked to a release.
+
+The SDK will automatically manage the start and end of the sessions when the SDK is initialized.
```

Comment:
I realize you just moved the content but since not all SDKs can start/end sessions automatically and the way they work varies a bit, maybe reword somehow to be more generic?

```suggestion
Most Sentry SDKs can manage sessions automatically. For example, mobile SDKs can end session when the app goes to the background for longer than 30 seconds. Other SDKs end the session when the process exit.
```

Reply:
I also feed the original sentence a little bit confusing. Maybe we just need to tell people "The SDK will automatically send session information" So they know that sessions are recorded, when exactly they start and stop is imho not that important here. What do you think?

Reply:
I have now applied your suggestion, but removed the examples, because a few paragraphs below is described what a session is. I think the description of what is a session, should be done in the platform specific sections.

Reply:
@bruno-garcia - can you take another look, please?